python/cuml/fil/fil.pyx

```diff
@@ -296,7 +309,13 @@ cdef class ForestInference_impl():
                                         float threshold,
                                         str storage_type):
         cdef treelite_params_t treelite_params
-        treelite_params.output_class = output_class
+
+        self.output_class = output_class
```

Comment:
How do we know if a classification model is being loaded? The only way we can find it out is the `output_class` parameter.

Reply:
a self variable for `output_class` is created to pass on the information to other functions. ex. the predict function where we check if the user is trying to use predict_proba for regression model.

Reply:
This does help, e.g. to ensure that `predict_proba()` is only called for classification models.

However, there is no need for a warning if `output_class==False`: it's fine for regression models.